src/index.js

```diff
@@ -4,6 +4,7 @@ import TodoItem from './modules/TodoItem.js';
 
 const form = document.querySelector('form');
 const todoWrapper = document.querySelector('.todo-container');
+const removeCompleted = document.querySelector('button');
```

Comment:
- [ ] [OPTIONAL] Please try to add at least 5 commits to per repo next time to make ti loo professional 👍🏼😃